### PR TITLE
Final mouse hover

### PR DESCRIPTION
**Pull request by alexli123**
The hand selector is now everywhere that is clickable.

### CODE REVIEW

File: bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 import java.util.Map;
 
+import javax.mail.Message;
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java]: If those changes are part of a later pull request, they don't need to be submitted in this one. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java [hunk 2]
```diff
@@ -11,6 +15,7 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import de.fraunhofer.abm.app.EmailConfigInterface;
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java]: If those changes are part of a later pull request, they don't need to be submitted in this one. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java [hunk 3]
```diff
@@ -27,6 +32,9 @@
 	private UserDao userDao;
 	@Reference
 	private UserAdmin userAdmin;
+	
+	@Reference
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java]: If those changes are part of a later pull request, they don't need to be submitted in this one. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java [hunk 4]
```diff
@@ -46,6 +54,14 @@ public String getApproval(RESTRequest rr) {
 			user.getCredentials().put("password", password);
 			Group registeredUserGroup = (Group) userAdmin.getRole("RegisteredUser");
 			registeredUserGroup.addMember(user);
+			/**String sbj = params.get("username") + " Approved on ABM";
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java]: This change is already part of another pull request. Please don't mix both.

Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java]: Why are those changes commented out? If they are part of a later pull request, they don't need to be submitted in this one. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/template/home.htm [hunk 5]
```diff
@@ -25,7 +25,7 @@ <h2>Private Collections</h2>
 						<!-- <th>Size</th> -->
 						<th>ID</th>
 					</tr>
-					<tr ng-repeat="row in userCollections">
+					<tr style="cursor:pointer;" ng-repeat="row in userCollections">
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/template/home.htm]: This change are part of another pull request. Don't mix both. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/template/home.htm [hunk 6]
```diff
@@ -64,7 +64,7 @@ <h3>Pinned Collections</h3>
 					<th>ID</th>
 					<th></th>
 				</tr>
-				<tr ng-repeat="row in pinned">
+				<tr style="cursor:pointer;" ng-repeat="row in pinned">
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/template/home.htm]: This change are part of another pull request. Don't mix both. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/template/home.htm [hunk 7]
```diff
@@ -118,7 +118,7 @@ <h2>Loading...</h2>
 				<th>ID</th>
 				<th ng-hide="user === undefined"></th>
 			</tr>
-			<tr ng-repeat="row in publicData">
+			<tr style="cursor:pointer;" ng-repeat="row in publicData">
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/template/home.htm]: This change are part of another pull request. Don't mix both. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/web/style.css [hunk 8]
```diff
@@ -19,11 +19,13 @@ body {
 
 .custom-right{
 	position: fixed;
-	right: 15px;
-	
-	
-	
+	right: 15px;
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/web/style.css]: Those changes are part of another pull request. Don't mix both. Please remove.

File: docs/abm.sql [hunk 9]
```diff
@@ -1,152 +0,0 @@
-CREATE TABLE `build_result` (
```
Comment (nguyenLisa) [inline docs/abm.sql]: This file should not be removed. Please add again.

File: bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html [hunk 10]
```diff
@@ -46,7 +46,7 @@
 				<li ng-class="{active: page() == '/login'}" ng-show="user === undefined"><a href="#/login">Login</a></li>
 				<li ng-hide="user === undefined"><a href="" ng-click="logout()">Logout</a></li>
 			</ul>
-		<h3 class="text-muted custom-right">Automatic Benchmark Management</h3>
+		<h3 class="text-muted custom-right"><br class="rwd-break">Automatic Benchmark Management</h3>
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html]: Those changes are part of another pull request. Don't mix both. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/web/style.css [hunk 11]
```diff
@@ -19,11 +19,13 @@ body {
 
 .custom-right{
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/web/style.css]: Those changes are part of another pull request. Don't mix both. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/web/style.css [hunk 12]
```diff
@@ -19,11 +19,13 @@ body {
...
 	position: fixed;
-	right: 15px;
-	
-	
-	
+	right: 15px;	
 }
+
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/web/style.css]: Those changes are part of another pull request. Don't mix both. Please remove.

File: bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html (hunk 10 above)
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html]: Those changes are part of another pull request. Don't mix both. Please remove.

**Comment by alexli123** [comment]
I think this is up to date and covers both the mouse hover + title issues. Please let me know if there are any issues.

File: bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html [hunk 13]
```diff
@@ -47,9 +47,7 @@
 				<li ng-class="{active: page() == '/login'}" ng-show="user === undefined"><a href="#/login">Login</a></li>
 				<li ng-hide="user === undefined"><a href="" ng-click="logout()">Logout</a></li>
 			</ul>
-
-		<h3 class="text-muted custom-right title-text">Automated Benchmark Management</h3>
-
+		<h3 class="text-muted custom-right"><br class="rwd-break">Automatic Benchmark Management</h3>
```
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/static/de.fraunhofer.abm/index.html]: "Automated", not "Automatic"

File: docs/abm.sql [hunk 14]
```diff
@@ -1,13 +1,3 @@
...
```
Comment (nguyenLisa) [inline docs/abm.sql]: Why is this table removed?

File: bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java (hunk 1 above)
Comment (nguyenLisa) [inline bnd-workspace/de.fraunhofer.abm.app/src/de/fraunhofer/abm/app/controllers/UserApprovalController.java]: If those changes are part of a later pull request, they don't need to be submitted in this one. Please remove the from this pull request.

**Comment by alexli123** [comment]
Can you please check if I've hit all the comments you've made - thanks!

**Comment by nguyenLisa** [comment]
You still have unused imports in UserApprovalController.java. Please fix.

**Comment by alexli123** [comment]
My mistake - to be quite honest I'm not sure how they got there. Please let me know if I've deleted all the unused ones.

**Pull request merged**